recipes/sqlpp11-connector-sqlite3/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
   "0.29":
     url: "https://github.com/rbock/sqlpp11-connector-sqlite3/archive/0.29.tar.gz"
-    sha256: 2904ddd578adb0e122a70f73a80bae4843434d685b7a23ae3ab43471c35efc4f
+    sha256: "2904ddd578adb0e122a70f73a80bae4843434d685b7a23ae3ab43471c35efc4f"
+  "0.30":
```

Comment:
Hey @SpaceIm 
In my latest recipes, I'm adding the most recent versions at the top.
That makes making changes easier, imho.
What do you think?

Reply:
I was doing the same things in the past, then I saw some automated script of @SSE4 (not sure) running and reordering conandata.yml and config.yml from older to latest. But I agree that latest first makes more sense.

Reply:
Maybe @Croydon 's @CroydonBot

Reply:
My scripts aren't (yet) capable of adding new versions. That was probably @SSE4's scripts for adding new versions

But I don't think that it matters either way if it is ascending or descending as long as it is ordered within a recipe

Reply:
yes, it was mine script. the order was automatic by `yaml` python module.